packages/embedded-provider-core/src/auth/jwt-auth.ts

```diff
@@ -84,7 +83,6 @@ export class JWTAuth {
         walletId: result.walletId,
         organizationId: result.organizationId,
         provider: "jwt",
-        userInfo: result.userInfo || {},
```

Comment:
Why are we removing `customAuthData` and `userInfo`? They're still referenced in `packages/embedded-provider-core/README.md` and some of the loosely typed mocks.